changelog.html

```diff
@@ -43,6 +43,12 @@
 <h1>
 Monitoring Plugin Changelog
 </h1>
+    
+<p><b>1.8.2</b> -- November 25, 2019</p>
+<ul>
+    <li>[<a href='https://github.com/igniterealtime/openfire-monitoring-plugin/issues/60'>Issue #53</a>] - Java11 ClassNotFoundException when deploying.</li>
```

Comment:
This adds issue 53 to the changelog (while providing a link to issue 60), which isn't part of this PR. I'll merge the PR as-is, and manually correct this later.